test/integration/dnet/helpers.bash

```diff
@@ -168,7 +168,7 @@ EOF
 	   -v $(pwd)/${TMPC_ROOT}:/scratch \
 	   -v /usr/local/bin/runc:/usr/local/bin/runc \
 	   -w /go/src/github.com/docker/libnetwork \
-	   mrjana/golang ./cmd/dnet/dnet -d -D ${hopt} -c ${tomlfile}
+	   mrjana/golang ./bin/dnet -d -D ${hopt} -c ${tomlfile}
```

Comment:
I meant to ask, what's in `mrjana/golang`?
Can we replace with the `libnetwork-build` image?